.drone.yml

```diff
@@ -0,0 +1,30 @@
+workspace:
+  base: /go
+  path: src/github.com/crusttech/crust
+
+kind: pipeline
+name: crust
+
+steps:
+- name: build
+  image: golang:1.11-alpine
+  environment:
+    CGO_ENABLED: 0
+    GOOS: linux
+    GOARCH: amd64
+  commands:
+  - apk add --no-cache git
+  - go build -ldflags "-X github.com/crusttech/crust/internal/version.BuildTime=`date +%FT%T%z` -X github.com/crusttech/crust/internal/version.Version=`git describe --always --tags`" -o build/crm-$GOOS-$GOARCH cmd/crm/*.go
+  - go build -ldflags "-X github.com/crusttech/crust/internal/version.BuildTime=`date +%FT%T%z` -X github.com/crusttech/crust/internal/version.Version=`git describe --always --tags`" -o build/messaging-$GOOS-$GOARCH cmd/messaging/*.go
+  - go build -ldflags "-X github.com/crusttech/crust/internal/version.BuildTime=`date +%FT%T%z` -X github.com/crusttech/crust/internal/version.Version=`git describe --always --tags`" -o build/system-$GOOS-$GOARCH cmd/system/*.go
+  - go build -ldflags "-X github.com/crusttech/crust/internal/version.BuildTime=`date +%FT%T%z` -X github.com/crusttech/crust/internal/version.Version=`git describe --always --tags`" -o build/system-cli-$GOOS-$GOARCH cmd/system-cli/*.go
+
+trigger:
+  ref:
+    - "refs/heads/**"
+    - "refs/pull/**"
+    - "refs/tags/**"
+  event:
```

Comment:
You should drop the event, all these events are already covered by ref

Reply:
Thanks